Client/Classes/Handlers/ClientSquadHandler.sqx

```diff
@@ -70,5 +70,28 @@ namespace Intrusion.Client
 				};
 			};
 		};
+		
+		// Change unit uniform.
+		// _unit (Object): The unit to change dress on.
+		// _role (String): Role in vehicle.
+		// _vehicle (Object): The vehicle that the unit entered.
+		public static method SetUniform(
+			"_unit" as Object,
+			"_role" as String,
+			"_vehicle" as IntrusionVehicle)
+		{
+			private ["_professionType" as ProfessionType, "_uniformItems" as Array, "_uniformInfo" as UniformInfo];
+			_professionType = _vehicle.OwnerProfession;
+			_uniformItems = uniformItems _unit;
+			
+			_uniformInfo = [side _unit, _professionType, [_role] call VehicleRoleMeta.fromString] call gCommon.UniformConfig.GetUniformInfo;
```

Comment:
Use of global variable. Follow pattern by adding class variable "_mUniformConfig" instead and call _mUniformConfig.GetUniformInfo.